content/en/blog/2022/exponential-histograms.md

```diff
@@ -0,0 +1,297 @@
+---
+title: "OpenTelemetry Exponential Histograms: Better Data, Zero Configuration"
+linkTitle: OpenTelemetry Exponential Histograms
+date: 2022-08-22
+author: "[Jack Berg](https://github.com/jack-berg)"
+canonical_url: https://newrelic.com/blog/best-practices/opentelemetry-histograms
+---
+
+Histograms are a powerful tool in the observability tool belt. Leading
+instrumentation projects like OpenTelemetry and Prometheus support histograms
+because of their ability to efficiently capture and transmit distributions of
+measurements, enabling statistical calculations like percentiles (for example,
+p90, p95, p99).
+
+In practice, histograms come in several flavors, each with its own strategy for
+representing buckets and bucket counts. The first stable metric release for
+OpenTelemetry included explicit bucket histograms, and now OpenTelemetry is
+introducing a new exponential bucket histogram option. This exciting new format
+automatically adjusts buckets to reflect measurements and is more compressed to
+send over the wire. This blog post dives into the details of exponential
+histograms, explaining how they work, the problem they solve, and how to start
+using them now.
+
+## Intro to metrics in OpenTelemetry
+
+Before talking about exponential bucket histograms, let's do a quick refresher
+on some general OpenTelemetry metrics concepts. If you're already up to speed,
+skip ahead to [Anatomy of a histogram](#anatomy-of-a-histogram).
+
+Metrics represent aggregations of many measurements. We use them because it's
+often prohibitively expensive to export and analyze measurements individually.
+Imagine the cost of exporting the time of each request for an HTTP server
+responding to one million requests per second! Metrics aggregate measurements to
+reduce data volume and retain a meaningful signal.
+
+Like tracing (and someday soon logs), OpenTelemetry metrics are broken into
+the [API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md)
```

Comment:
Here and elsewhere in the file, link to the local spec pages under [/docs/reference/specification/](https://opentelemetry.io/docs/reference/specification/). For example, the link on this line would become `/docs/reference/specification/metrics/api/`.